src/objects/Attribute__c.object

```diff
@@ -220,17 +280,65 @@
                 <fullName>Accommodation</fullName>
                 <default>false</default>
             </values>
+            <values>
+                <fullName>Asylee</fullName>
+                <default>false</default>
+            </values>
+            <values>
+                <fullName>Economically Disadvantaged</fullName>
+                <default>false</default>
+            </values>
+            <values>
+                <fullName>English Learners</fullName>
+                <default>false</default>
+            </values>
             <values>
                 <fullName>Family Consideration</fullName>
                 <default>false</default>
             </values>
+            <values>
+                <fullName>Foster Care</fullName>
+                <default>false</default>
+            </values> 
+            <values>
+                <fullName>Free Lunch</fullName>
+                <default>false</default>
+            </values>
+            <values>
+                <fullName>Homeless</fullName>
+                <default>false</default>
+            </values> 
+            <values>
+                <fullName>Immigrant</fullName>
+                <default>false</default>
+            </values>
             <values>
                 <fullName>International Student</fullName>
                 <default>false</default>
             </values>
             <values>
                 <fullName>Other</fullName>
                 <default>false</default>
+            </values> 
+            <values>
+                <fullName>Parent in Military </fullName>
```

Comment:
The space is here too

Reply:
Interesting. I saw those and deleted in my editor. I’ll clear those.